README.md

```diff
@@ -37,6 +37,19 @@ Usage / First steps
 First you should start the testing docker containers (see also next section of this README):
 
     vendor/bin/sdTest.sh start
+    
+    
+(De-)Activating plugin
```

Comment:
I think we should rename this because for activate and deactivate there a seperate commands available.
Why not to name it like 'add/remove plugin' and add a sentence about activating and deactivating the plugin with vendor/bin/sdPlugin.sh 71 activate / deactivate